src/features/game/state/GameStateManager.ts

```diff
@@ -504,6 +557,16 @@ class GameStateManager {
     return this.triggeredStateChangeActions;
   }
 
+  /**
+   * Return an array containing [string, number] pairs
+   * representing quizzes and the corresponding scores.
+   *
+   * @returns {[string, number][]}
+   */
+  public getQuizScores(): [string, number][] {
+    return [...this.quizScores];
+  }
```

Comment:
Ditto.

Reply:
The method is used in GameGlobalAPI, GameDialogueManager and GameActionConditionChecker